utilities/tools/helper/Get-SpecsAlignedResourceName.ps1

```diff
@@ -69,6 +69,7 @@ function Get-SpecsAlignedResourceName {
 
     $rawProviderNamespace, $rawResourceType = $reducedResourceIdentifier -Split '[\/|\\]', 2 # e.g. 'keyvault' & 'vaults/keys'
 
```

Comment:
Should we use the word "ApiSpec" instead of "Spec"? I think we need to align the naming between the settings flag, the scripts parameters and this cmdlet name. 

E.g: 

**settings**: publishUsingApiSpecName

**Scripts parameter**:
UseApiSpecName

**Cmdlet**:
Get-ApiSpecResourceName.ps1

Reply:
I kept them actually different on purpose but can align them if you think it confuses users

Reply:
Yes please. I thought at first spec was related to template specs 😅 but might be confusing for others if they're debugging

Reply:
Updated